tests/qgis-projects/tests/replace_database_connection_qgs_project.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+if [ "$#" -ne 6 ]; then
+    echo "Bad numbers of arguments"
+    exit 1
```

Comment:
```suggestion
    echo "Bad numbers of arguments"
    echo "This script is to replace the database connection in a set of QGIS projects."
    echo "replace_database_connection_qgs_project.sh QGS_DIRECTORY HOST DB_NAME USER PASSWORD PORT"
    echo "Be careful, original files will be updated"
    exit 1
```

An help within the script was enough I think. It's usually what is done for a bash script, because only geeks (from 3Liz) will execute the script ;-)

Exactly like if you write `docker-compose` without parameter in your terminal, it will just display some help.